app/app.js

```diff
@@ -34,6 +39,31 @@ env.addFilter('componentNameToMacroName', helperFunctions.componentNameToMacroNa
 // Set view engine
 app.set('view engine', 'njk')
 
+// middleware to compile component scss on request
+const destPath = configPaths.public
+app.use(sassMiddleware({
+  src: root,
+  response: false,
+  dest: destPath,
+  outputStyle: 'extended'
+}))
+
+// get the above request and tranform css
+app.use(postcssMiddleware({
+  plugins: [
```

Comment:
I think we should have a single source of truth for how we build our files, this would be really easy to get out of sync with what we ship to users.

Reply:
created a single place for the plugins